src/manifest/dbtProject.ts

```diff
@@ -374,7 +374,7 @@ export class DBTProject implements Disposable {
 ),
 renamed as (
     select
-        ${columnsInRelation.map((column) => column.column).join(",\n        ")}
+        ${columnsInRelation.map((column) => `"${column.column}"`).join(",\n        ")}
```

Comment:
```suggestion
        ${columnsInRelation.map((column) => `{{ adapter.quote("${column.column}") }}`).join(",\n        ")}
```

Please also test this as I didn't try it myself.